dev-proxy/ProxyHost.cs

```diff
@@ -268,6 +270,26 @@ public ProxyHost()
             Arity = ArgumentArity.ZeroOrMore
         };
         _urlsToWatchOption.AddAlias("-u");
+        
+        _timeoutOption = new Option<long?>(TimeoutOptionName, "Time in seconds after which the proxy quits, resets when a request is being made")
+        {
+            ArgumentHelpName = "inactivityTimeout",
```

Comment:
Let's make it consistent with other args

```suggestion
            ArgumentHelpName = "timeout",
```